documental/views.py

```diff
@@ -22,7 +22,7 @@ class AuthModelMix:
 class ActionListView(AuthModelMix, generics.ListAPIView):
     """Returns the list of data in `models.DocsAction`."""
 
```

Comment:
Trailing whitespaces.

Use pep8 to format this document.

Reply:
.